examples/music.py

```diff
@@ -4,11 +4,59 @@
 import discord
 from discord.ext import commands
 import lavalink
-from lavalink.audio_events import DefaultEventAdapter
+from lavalink.audio_events import TrackResumeEvent, TrackExceptionEvent, TrackStartEvent, TrackPauseEvent, TrackEndEvent, \
+    TrackStuckEvent, AudioTrack, AbstractPlayerEventAdapter
 
 time_rx = re.compile('[0-9]+')
 
 
+class DefaultEventAdapter(AbstractPlayerEventAdapter):
+    """
+    The default event adapter
+    TODO: Add more shit to this class
+    """
+
+    def __init__(self, player, ctx):
+        super().__init__(player)
+        self.ctx = ctx
+        self.bot = ctx.bot
+        self.queue = []
+
+    async def track_resume(self, event: TrackResumeEvent):
+        pass
+
+    async def track_exception(self, event: TrackExceptionEvent):
+        pass
+
+    async def track_start(self, event: TrackStartEvent):
+        track = event.track
```

Comment:
If we store player in the event instead, users will have more freedom in what they can access. (Users may store custom data within players using `store` and `fetch` functions). They can also use `player.current` instead of `event.track` to access the track.